### ISSUE TITLE
spyder.ini problems on Windows7

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#715]
_From [mark...@gmail.com](https://code.google.com/u/115613240999600774336/) on 2011-07-25T09:05:18Z_

Hello List,

We are experiencing the same problems with the spyder.ini file as has been reported in a few other posts. We are running PythonXY on Windows7. Two students on two different machines got very similar problems a week apart. The spyder.ini file is corrupted and seems to end somewhere arbitrarily. I edited the spyder.ini file by hand so that it didn't have any errors, and that worked (at least for now). Anybody any idea how the spyder.ini file can get corrupted? Any fixes?

Thanks, Mark

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=715_

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2011-07-25T17:27:30Z_

Hi Mark,

Thanks for reporting this problem. I'll look at it tommorrow, I hope I can come up with a solution too. Could you tell me your Spyder version please?

Cheers,
Carlos

PS. Just for the record, I leave here the link to the post in google groups: http://groups.google.com/group/spyderlib/browse_thread/thread/15a91a5d10714ee3

**Labels:** -Priority-Medium Priority-Critical Cat-Settings

**Comment by spyder-issues-migrator** [comment]
_From [mark...@gmail.com](https://code.google.com/u/115613240999600774336/) on 2011-07-26T01:30:18Z_

Sorry, Carlos, but I don't know the Spyder version.
Both students reinstalled PythonXY on their machines. The spyder version
that went wrong was installed with pythonxy sometime this Spring.
Strangely enough, that didn't fix it.
And I have no idea how to duplicate the problem.
A very similar thing happened to both students, about a week apart. The
spyder.ini file was corrupted, which mean that part of it was missing.
Different parts on different machines. Reinstalling Pythonxy didn't solve
anything, strangely enough. I edited the spyder.ini file such that it didn't
have any syntax errors. And then Spyder ran fine. I looked on googlecode,
but couldn't locate the correct spyder.ini file in the repository. Is that
generated on the fly?
Thanks for your help. Let me know if I can help in any way,
Mark
